protostar/self/cache_io_test.py

```diff
@@ -21,6 +21,9 @@ def test_cache_simple(tmp_path):
 
     assert cache_io.read(cache_name) is None
 
+    # pylint: disable=protected-access
+    assert cache_io._gitignore_path.read_text(encoding="utf-8") == "*\n"
```

Comment:
I think it would be also nice to verify that modification timestamp of the file does not change during subsequent writes.

Reply:
[check if the gitignore is not modified when already created](https://github.com/software-mansion/protostar/pull/973/commits/c62c8ddcce7665e812bccb440f87331c811d407d)